src/apollo/server.js

```diff
@@ -0,0 +1,34 @@
+const { ApolloServer } = require('apollo-server-express');
```

Comment:
I think I might want to move this into the middleware folder for now. It doesn't make sense to have a dedicated Apollo folder right now.